packages/hardware-trezor/src/transformers/additionalWitnessRequests.ts

```diff
@@ -0,0 +1,23 @@
+import * as Trezor from 'trezor-connect';
+import { CardanoKeyConst, GroupedAddress, util } from '@cardano-sdk/key-management';
+import { TrezorTxTransformerContext } from '../types';
+import { isNotNil } from '@cardano-sdk/util';
+import concat from 'lodash/concat';
+import uniq from 'lodash/uniq';
+
+export const createRewardAccountKeyPath = (knownAddress: GroupedAddress) => [
```

Comment:
Should that belong to `keyPaths.ts`? (just thinking out loud)

Reply:
@marcin-mazurek check the comment https://github.com/input-output-hk/cardano-js-sdk/pull/851#discussion_r1286943001 I think it answers your question